src/server/index.ts

```diff
@@ -393,16 +399,18 @@ export class DocumentDriveServer implements BaseDocumentDriveServer {
         }
     }
 
-    registerListener(input: CreateListenerInput): Promise<Listener> {
-        throw new Error('Method not implemented.');
+    registerListener(
+        input: CreateListenerInput
+    ): Promise<Listener> {
+        return this.storage.registerListener(input);
```

Comment:
Since we are passing the storage to `ListenerManager` we can just call this in the `addListener`. This way `DocumentDriveServer` doesn't need to know how the listeners are stored.